tsdb/index/inmem/meta.go

```diff
@@ -286,7 +286,10 @@ func (m *Measurement) Rebuild() *Measurement {
 	// expunged.  Note: we're using SeriesIDs which returns the series in sorted order so that
 	// re-adding does not incur a sort for each series added.
 	for _, id := range m.SeriesIDs() {
-		nm.AddSeries(m.SeriesByID(id))
+		s := m.SeriesByID(id)
```

Comment:
nit, you can put this on one line `if s := m.SeriesByID(id); s != nil {`

Reply:
Also, if we don't want to add `nil` Series to the new measurement can't we just add the no-op for the `nil` case to the `AddSeries` implementation?

Reply:
Yeah, I can have `AddSeries` handle adding `nil`.